tests/unit/decorators/local-storage-test.js

```diff
@@ -159,4 +160,27 @@ module('Unit | Decorator | @localStorage', function (hooks) {
       assert.equal(klass.foo, 'bar');
     });
   });
+
+  module('test support', function () {
+    test('developer can reinitialize a local storage key in tests', function (assert) {
+      // create a class, which will be used between multiple tests runs
+      class Foo {
+        @localStorage foo;
+      }
+
+      // use the class in first test
+      assert.equal(new Foo().foo, null);
+
+      // reset local storage cache as developer is expected to do in beforeEach hook
+      window.localStorage.clear();
+      clearLocalStorageCache();
```

Comment:
```
      // reset local storage cache as developer is expected to do in beforeEach hook
      window.localStorage.clear();
      clearLocalStorageCache();
```
Is the code above needed at all? We are clearing the cache in beforeEach hook and after with code below:
```
      window.localStorage.setItem('foo', JSON.stringify('bar'));
      initalizeLocalStorageKey('foo');
```
Code won't hurt but I believe it is not needed.

Reply:
This test simulates the code we expect a consumer to run in two tests. Therefore the cache is reset in between the two simulated tests as we ask the user to do in documentation.